contributors/contributors/community.md

```diff
@@ -103,45 +86,45 @@ All contributors Level 2 and above are eligible for a [custom-printed Mattermost
 
 Community Coordinator: Jason Blais and Ben Schumacher
 
-#### [Translation Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
+#### Translation Contributors
 
 Translation Contributors can create and update translations to existing or new Mattermost languages at [https://translate.mattermost.com](https://translate.mattermost.com/). In addition, community members can become “Language owners” and help review, coordinate and supervise the translation of Mattermost into languages they speak.
 
 Community Coordinator: Jason Blais
 
-#### [Documentation Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
+#### Documentation Contributors
 
 Documentation Contributors update, edit, and create new documentation, help text, training, and other materials to help the community get the most out of Mattermost software. Core documentation is found at [https://docs.mattermost.com](https://docs.mattermost.com/) and contributions can be made via the open source repository at [https://github.com/mattermost/docs/](https://github.com/mattermost/docs/).
 
 Community Coordinator: Jason Blais and Justine Geffen
 
-#### [Quality Assurance Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
+#### Quality Assurance Contributors
 
 Quality Assurance Contributors help find, confirm and report product defects via our process at [https://www.mattermost.org/filing-issues/](https://www.mattermost.org/filing-issues/). Anyone who encounters a verifiable defect is welcome to file a bug on the system.
```

Comment:
Confirming with the QA team that this is the most up to date link for filing bugs.

Reply:
Wondering if we should link to the Bugs channel too?

Reply:
We could link to the Bugs channel, but it's behind email login credentials -- so in that regard GitHub issues is often simpler for QA contributors.

Reply:
That makes sense, we can disregard this suggestion. QA have confirmed that the contents in the link provided are correct, so resolving this comment.